src/logic/safe/store/actions/utils.ts

```diff
@@ -94,6 +95,7 @@ export const extractRemoteSafeInfo = async (remoteSafeInfo: SafeInfo): Promise<P
   safeInfo.collectiblesTag = remoteSafeInfo.collectiblesTag
   safeInfo.txQueuedTag = remoteSafeInfo.txQueuedTag
   safeInfo.txHistoryTag = remoteSafeInfo.txHistoryTag
+  safeInfo.chainId = remoteSafeInfo.chainId as ETHEREUM_NETWORK
```

Comment:
Do you need to add `chainId` to the `SafeInfo` type? I don't get why you needed to cast this.